pkg/sync/v14/data/Deployment.apps/openshift-azure-logging/omsagent-rs.yaml

```diff
@@ -54,11 +54,11 @@ spec:
           name: in-rs-tcp
         resources:
           limits:
-            cpu: 150m
-            memory: 600Mi
+            cpu: 1
+            memory: 740Mi
           requests:
-            cpu: 75m
```

Comment:
just check when you run multiple sync's that it doesn't keep changing this. when I did this, i  remember there was..

Reply:
I believe it is not an issue this time, because it is not updating the same deployment, but completely replacing it. At least after `make sync-run` this is what I can see:
```
[ openshift-azure (omsagent) ] k get deploy omsagent-rs -n openshift-azure-logging -o yaml | grep -A 6 resources 
        resources:
          limits:
            cpu: "1"
            memory: 740Mi
          requests:
            cpu: 150m
            memory: 250Mi
```

Reply:
@asalkeld I found an issue with limit 1 - it does seem to try to update it every run due to types: 
```
                                                                "limits": map[string]interface{}{                                                                                                                                                                                                                                                          
-                                                                       "cpu":    string("1"),                                                                                                                                                                                                                                                             
+                                                                       "cpu":    int64(1),                                                                                                                                                                                                                                                                
                                                                        "memory": string("740Mi"),                                                                                                                                                                                                                                                         
                                                                }, 
```
I have changed that limit to 950 instead there. 
But as for this line that you've commented - I don't see any problem. On the second run (without any changes), sync-pod did not try to perform any changes

Also noticed 740Mi, I'll fix this typo